### DISCUSSION TITLE
"Another instance is already running" - After installation

### DISCUSSION THREAD

**Discussion by twoandthree** [kangyu-california/PersistentWindows#196] [Q&A]
Can't seem to open the program after installing. I've tried running it as an administrator, but no luck. Any advice would be appreciated. Thank you!

**Comment by kangyu-california** [comment]
For manual reinstall, you may need to kill old instance of PersistentWindows in task manager first.
![image](https://user-images.githubusercontent.com/59128756/209031323-a5166f21-4d90-42d7-be82-238b189faf25.png)
